Simplenote.xcodeproj/project.pbxproj

```diff
@@ -6180,6 +6145,14 @@
 				minimumVersion = 0.3.0;
 			};
 		};
+		3FA8817C2DFAADD200E1F4A3 /* XCRemoteSwiftPackageReference "ZIPFoundation" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/weichsel/ZIPFoundation";
+			requirement = {
+				kind = exactVersion;
+				version = 0.9.15;
```

Comment:
I used exact to match the version resolved in `Podfile.lock` and avoid introducing changes because of fetching a newer version (at the time of writing, the latest is 0.9.19).

We can always follow up with a dedicated PR to update.